android/src/main/java/com/brentvatne/react/VideoDecoderPropertiesModule.kt

```diff
@@ -67,6 +67,6 @@ class VideoDecoderPropertiesModule(reactContext: ReactApplicationContext?) : Rea
     companion object {
         private val WIDEVINE_UUID = UUID(-0x121074568629b532L, -0x5c37d8232ae2de13L)
         private const val SECURITY_LEVEL_PROPERTY = "securityLevel"
-        private const val REACT_CLASS = "VideoDecoderProperties"
```

Comment:
is it really necessary to change the module name ?

Reply:
No, it's not required, but I changed it to reduce the chance of duplicates.